packages/widget_catalog/lib/main.dart

```diff
@@ -30,7 +30,8 @@ class MainApp extends StatelessWidget {
           ],
         ),
         TextScaleAddon(
-          scales: [1.0, 2.0],
+          min: 0.6,
+          max: 1.9,
```

Comment:
The default value would be omitted, so we dared to use a value that is not close to the end of the list!